### ISSUE TITLE
loss is between 3~4 and does not decrease any more

### ISSUE THREAD

**Issue by xuyunlu1030** [facebook/C3D#351]
I use your ResNet18-C3D finetuing demo from Sports-1M pretrained model without changing any parameter or data. The train loss is decreasing from 4.8 to around 3.0 and does not decrease any more, and the corresponding training acc is around 30% ,val acc is around 12%. 
Is there something WRONG with my training process or parameter setting?

**Comment by zheng123789** [comment]
The same problem as you. Looking forward the solution.

**Comment by zzy123abc** [comment]
I Train from scratch,and I meet the same problem.Did you train from scratch or fitune.?

**Comment by dutran** [comment]
Just tried running myself, after 280 iter, i get

I0129 21:49:59.901010 3548283 solver.cpp:238]     Train net output #0: loss = 2.48471 (* 1 = 2.48471 loss)
I0129 21:49:59.901021 3548283 sgd_solver.cpp:105] Iteration 220, lr = 0.001
I0129 21:50:42.194798 3548283 solver.cpp:219] Iteration 240 (0.472888 iter/s, 42.2933s/20 iters), loss = 2.59051
I0129 21:50:42.194955 3548283 solver.cpp:238]     Train net output #0: loss = 2.59051 (* 1 = 2.59051 loss)
I0129 21:50:42.194968 3548283 sgd_solver.cpp:105] Iteration 240, lr = 0.001
I0129 21:51:24.477214 3548283 solver.cpp:219] Iteration 260 (0.473017 iter/s, 42.2818s/20 iters), loss = 2.13765
I0129 21:51:24.477304 3548283 solver.cpp:238]     Train net output #0: loss = 2.13765 (* 1 = 2.13765 loss)
I0129 21:51:24.477326 3548283 sgd_solver.cpp:105] Iteration 260, lr = 0.001
I0129 21:52:06.768424 3548283 solver.cpp:219] Iteration 280 (0.472917 iter/s, 42.2907s/20 iters), loss = 2.20005
I0129 21:52:06.768625 3548283 solver.cpp:238]     Train net output #0: loss = 2.20005 (* 1 = 2.20005 loss)
I0129 21:52:06.768637 3548283 sgd_solver.cpp:105] Iteration 280, lr = 0.001

**Comment by nigegedck** [comment]
@xuyunlu1030 I used the c3d_resnet18_sports1m_r2_iter_2800000.caffemodel to finetuing the C3D v1.1 demo. I meet the same problem.
Do you have solved the problem?

**Comment by liulinyi007** [comment]
@dutran @xuyunlu1030 @nigegedck I met the same problem. Do you know how can solve it?
i used c3d_resnet18_sports1m_r2_iter_2800000.caffemodel, used ffmpeg to extract the frm(add "-q:v 1")
THXXX!

**Comment by liulinyi007** [comment]
I solved it. I comment USE_CUDNN := 1, and then it's ok now.
